src/features/Classes/data/slice.js

```diff
@@ -37,6 +37,15 @@ export const classesSlice = createSlice({
     updateFilters: (state, { payload }) => {
       state.filters = payload;
     },
+    fillClassesTable: (state, { payload }) => {
+      state.table.data = payload;
```

Comment:
If this is not information for a table as far I can see, please store it in another part